raiden/storage/sqlite.py

```diff
@@ -1472,16 +1473,18 @@ def get_latest_light_client_non_closing_balance_proof(self, channel_id, non_clos
                    balance_proof,
                    lc_balance_proof_signature
             FROM light_client_balance_proof
-            WHERE channel_id  = ? AND sender = ?
+            WHERE channel_id  = ? AND sender = ? and token_network_address = ?
```

Comment:
`and` should be `AND` for consistency's sake.

not a blocker

Reply:
fixed